python/openmldb_tool/diagnostic_tool/diagnose.py

```diff
@@ -44,6 +45,11 @@
     False,
     "If set, all server in config file will be treated as local server, skip ssh.",
 )
+flags.DEFINE_string(
+    "db",
+    "",
+    "Specify databases to diagnose, split by ','.",
```

Comment:
It's ok to add 'only inspect online works' in help str. To avoid user use 'inspect offline --db' or 'test--db', we don't support it now.

Reply:
OK